app/scripts/metamask-controller.test.js

```diff
@@ -27,6 +27,9 @@ import {
 import {
   RatesController,
   TokenListController,
+  MultichainBalancesController,
+  BalancesTracker as MultichainBalancesTracker,
```

Comment:
Kinda unrelated to this, but I guess we should rename `BalancesTracker` to `MultichainBalancesTracker` someday, since those 2 are very related to each other, WDYT?

Reply:
Ok, lets open a small ticket to change it in the packages

Reply:
https://github.com/MetaMask/accounts-planning/issues/805